store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10390,4 +10390,7 @@ TODO: delete them permanently from here
   <desc>Whether retention policy feature is enabled</desc>
 </attr>
 
+<attr id="4102" name="zimbraAliasListToHide" type="email" max="256" immutable="1" cardinality="multi" optionalIn="mailRecipient" since="10.1.0">
+  <desc>aliases list to be hidden in autocomplete</desc>
```

Comment:
Add more details about how the values are set.
zimbraMailLocalBind is described as follows, for example.
```
<attr id="1380" name="zimbraMailLocalBind" type="boolean" cardinality="single" immutable="1" optionalIn="server" requiresRestart="mailbox" since="8.0.0">
  <desc>Specifies whether the http server should bound to localhost or not. This is an immutable property and is generated based on zimbraMailMode and zimbraMailBindAddress.</desc>
</attr>
```